src/Kunstmaan/PagePartBundle/PageTemplate/PageTemplateConfigurationParser.php

```diff
@@ -65,20 +64,45 @@ public function parse($name)
     private function buildRegion($rawRegion)
     {
         $children = [];
+        $rows = [];
 
         if (isset($rawRegion['regions']) && count($rawRegion['regions'])) {
             foreach ($rawRegion['regions'] as $child) {
                 $children[] = $this->buildRegion($child);
             }
         }
 
+        if (isset($rawRegion['rows']) && count($rawRegion['rows'])) {
```

Comment:
instead of `if (isset() && count())` you can just set a default values:

```
$rawRegion = array_replace(['regions' => [], 'rows' => [], $rawRegion);
```